src/docs/asciidoc/en/firebirddocs/gfix/firebird-gfix.adoc

```diff
@@ -343,7 +343,9 @@ It is far quicker to read data from memory that it is to have to physically read
 
 The size of the database cache is dependent on the database page size and the number of buffers allocated, a buffer is the same size as a database page, and whether the installation is using Classic or Superserver versions of Firebird.
 
-In a Classic Server installation, each connection to the database gets its own relatively small cache of 75 pages while Superserver creates a much larger cache of 2,048 pages which is shared between all the connections.
+In a Classic and SuperClassic Server installation, by default, each connection to the database gets its own relatively small cache of 256 pages (was 75 before Firebird 3.0) while Superserver creates a much larger cache of 8196 pages (was 2048 before Firebird 3.0) which is shared between all the connections.
```

Comment:
Please add an entry in the version history table, and update the document header and license notic.